tests/loader/test_mobilebrick_loader.py

```diff
@@ -50,8 +50,8 @@ def test_get_image_full_res(self):
         with self.assertRaises(IndexError):
             self.loader.get_image_full_res(5)
 
-    def test_get_camera_intrinsics_full_res(self):
-        """Test the camera intrinsics at a given index."""
+    def test_get_gt_camera_intrinsics_full_res(self):
```

Comment:
we aren't calling `get_gt_camera_intrinsics_full_res()` in this fn though, right?

Bc we don't pass `use_gt_intrinsics: bool = True` in `setUp()`?